packages/gatsby-plugin-google-analytics/src/gatsby-browser.js

```diff
@@ -1,4 +1,35 @@
+function sendWebVitals() {
+  return import(/* webpackMode: "lazy-once" */ `web-vitals`).then(
```

Comment:
From https://webpack.js.org/api/module-methods/#magic-comments

> Note that this only makes sense in the case of a partially dynamic statement, e.g. import(`./locales/${language}.json`), where multiple module paths that can potentially be requested.

I don't fully understand `lazy-once` mode, but this note make it seems like it would same for this case as just `lazy` (default mode)?